tests/mgr/test_restful_operations_osd_endpoint.py

```diff
@@ -0,0 +1,116 @@
+import json
+
+from api.exceptions import ConfigError, OperationFailedError
+from api.restful.osd import Osd
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+LOG = Log(__name__)
+
+
+# Request header
+HEADER = {
+    "Accept": "application/vnd.ceph.api.v1.0+json",
+    "Content-Type": "application/json",
+}
+
+# Missing config parameter err msg
+CONFIG_ERR_MSG = "Mandatory config parameter '{}' not provided"
+
+
+def _get(config, url, endpoint, username, key, status_code):
+    # Check if any <args> are present
+    if config.get("args"):
+        endpoint += f'/{config.get("args")}'
+    # Check if the endpoint needs to have "command"
+    if config.get("command"):
+        endpoint += "/command"
+    # Use GET method on the /osd or or /osd/<arg> or /osd/<arg>/command
+    # endpoint of the RESTful module
+    _code, _ = Osd(url=url, api=endpoint).get(username=username, key=key, check_sc=True)
+    if _code == status_code:
+        LOG.info("RESTful API validation is successful")
```

Comment:
Not required, please remove from all the places.

Reply:
Done